nautobot_golden_config/migrations/0025_auto_20230912_0352.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2.20 on 2023-09-01 14:51
+# Generated by Django 3.2.20 on 2023-09-12 03:52
```

Comment:
I do not like that we are rewriting migration files at all.

Reply:
Perhaps I misunderstand, per Jeff we should not hand update and now you are saying we shouldn't rewrite them... I do not understand what the request is then? This is by my understanding the way it is done in core (and I have personally done many times) is to write a bunch of migrations, delete them, re-run migrations